usr/share/webapp-manager/firefox/profile/user.js

```diff
@@ -11,5 +11,4 @@ user_pref("toolkit.legacyUserProfileCustomizations.stylesheets", true);
 user_pref("browser.tabs.drawInTitlebar", false);
 user_pref("browser.tabs.inTitlebar", 0);
 user_pref("browser.contentblocking.category", "strict");
-user_pref("network.cookie.lifetimePolicy", 0);
-
+user_pref("privacy.clearOnShutdown.cookies", false);
```

Comment:
[This] is what LibreWolf customizes, so it probably makes more sense to just revert those prefs.

```suggestion
user_pref("privacy.sanitize.sanitizeOnShutdown", false);
user_pref("privacy.sanitize.timeSpan", 1);
```

Reply:
I will test this pref now, and I agree its better to just revert back the customization done by LibreWolf. Thanks for helping out :)

Reply:
@maltejur Looking at the page you linked to now, do you think we should also add this line?

`user_pref("privacy.clearOnShutdown.offlineApps", false);`

Reply:
Oh yeah of course. I misread that in the LibreWolf config and though that LibreWolf already sets that to false instead of true.

Reply:
From my testing so far everything works fine and the cookies aren't being cleared, so I've made a new commit with all the suggested changes. Thanks for your help 🙂